### PR TITLE
Rounded corners of `graduation-cap` icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?

- [ ] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Other: Optimization

### Description

Trying out https://lucide-studio.vercel.app

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
I wonder, should we not add rounding to the corners as in layers? 🤔

image

**Comment by jguddas** [comment]
> I wonder, should we not add rounding to the corners as in `layers`? 🤔
> 
> image

Round looks better to me.

File: icons/graduation-cap.svg [hunk 1]
```diff
@@ -9,6 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M22 10v6M2 10l10-5 10 5-10 5z" />
+  <path d="m2 10 10-5 10 5-10 5z" />
+  <path d="M22 10v6" />
   <path d="M6 12v5c3 3 9 3 12 0v-5" />
```
Comment (karsa-mistmere) [inline icons/graduation-cap.svg:14]: Here's the rounded version. Optimisation out the window. 😅 
```suggestion
  <path d="M12.83 5.18a2 2 0 0 0-1.66 0L2.6 9.08a1 1 0 0 0 0 1.832l8.58 3.91a2 2 0 0 0 1.66 0l8.58-3.9a1 1 0 0 0 0-1.832z" />
  <path d="M22 10v6" />
  <path d="M6 12.5V16a6 3 0 0 0 12 0v-3.5" />
```

**Review by ericfennis — APPROVED** [review]
Like this one better!

**Pull request merged**